.github/workflows/benchmark_regression_test.yml

```diff
@@ -0,0 +1,106 @@
+name: Benchmark Regression Check
+
+on:
```

Comment:
on workflow_dispatch would allow manual regression checks based on the current state of the branch.

Reply:
Done.